app/component/route/route-header.cjsx

```diff
@@ -2,24 +2,25 @@ React                 = require 'react'
 Icon                  = require '../icon/icon'
 GtfsUtils             = require '../../util/gtfs'
 Link                  = require 'react-router/lib/Link'
-
+classnames            = require 'classnames'
 
 class RouteHeader extends React.Component
```

Comment:
This is a larger question: Should we have somewhere explanation what props is supported by component?
Now there are "@props.className", "@props.trip", "@props.reverseId", "@props.route". It's not obvious what e.g. this means:
```
if @props.reverseId && !@props.trip
       reverse = <Link to="#{process.env.ROOT_PATH}linjat/#{@props.reverseId}">
           <Icon className={"route-header-direction-switch " + mode} img={'icon-icon_direction-b'}/>
         </Link>
```